New version: WamIPEDensity v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: WamIPEDensity
- Repository: https://github.com/Bourbon8464/WamIPEDensity.jl
- Created by: @Bourbon8464
- Version: v1.0.0
- Commit: 2acf7450436c977c83853860aba4e8dfb791e980
- Reviewed by: @Bourbon8464
- Reference: https://github.com/Bourbon8464/WamIPEDensity.jl/commit/2acf7450436c977c83853860aba4e8dfb791e980#commitcomment-172023643
- Description: Package for SpaceAGORA
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
- Working release for the WRS for SpaceAGORA. (Check ReadME for the full explanation)

[noblock]
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd676b685d22cf02300c3ac2191c1d593a6c521eec21c9621ea7af7d0559814c50d37357d8ba2a668031238ab3cfb79866ade0896a919eb5c848bdc5a44189de6fb314d78df7af22f4b9afa64ef6a2ce2b701b4d7f0dad4bbc9e40aea5f113d4d0c0c88e823c27514735cf79b9100916fd7b4c26dc3188c6952f9054bde5d751ded1f397100473558685455e4ab8e1493d388295eb4fbe8e4a9890d89227d196c304857cbb36678c15e66a99e5b623aa43f -->

Bourbon8464 commented:
@JuliaRegistrator register

changelog
Rapid increase in processing speed
Release notes:

Working release for the WRS for SpaceAGORA. (Check ReadME for the full explanation)

[noblock]

JuliaRegistrator committed:
New version: WamIPEDensity v1.0.0

UUID: b4be18c8-5d09-4dc7-9320-34e8e308bc8c
Repo: https://github.com/Bourbon8464/WamIPEDensity.jl.git
Tree: 06a2a3ea5704e60faef90b4e58b45fd829bfdb73

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
